.bazelrc

```diff
@@ -66,5 +55,5 @@ common --experimental_remote_cache_compression_threshold=2097152
 common --experimental_remote_merkle_tree_cache
 # Number of merkle trees to memoize (default 1000).
 common --experimental_remote_merkle_tree_cache_size=5000
-# Tells `xz` to use all available cores.
+
```

Comment:
Intentional?

Reply:
Nope! Just realized these got pulled in, fixing